tests/test_feature_selection.py

```diff
@@ -0,0 +1,101 @@
+# Authors: Soledad Galli <solegalli1@gmail.com>
+# License: BSD 3 clause
+import pytest
+import pandas as pd
+
+from feature_engine.feature_selection import DropFeatures
+
+
+def test_FeatureEliminator_initial(dataframe_vartypes, dataframe_na):
+    # test case 1: with dataframe_vartypes
+    transformer = DropFeatures(features_to_drop=["City", "dob"])
+    X = transformer.fit_transform(dataframe_vartypes)
+
+    # expected result
+    df = pd.DataFrame(
+        {'Name': ['tom', 'nick', 'krish', 'jack'],
+         'Age': [20, 21, 19, 18],
+         'Marks': [0.9, 0.8, 0.7, 0.6]}
+    )
+
+    # init params
+    assert transformer.features == ['City', 'dob']
+    # transform params
+    assert X.shape == (4, 3)
+    assert type(X) == pd.DataFrame
+    pd.testing.assert_frame_equal(X, df)
+
+
+def test_FeatureEliminator_non_existing_variables(dataframe_vartypes):
+    # test case 2: passing variables that doesn't exist
+    with pytest.raises(KeyError):
+        transformer = DropFeatures(features_to_drop="last_name")
+        X = transformer.fit_transform(dataframe_vartypes)
+
+
+def test_FeatureEliminator_non_existing_index(dataframe_vartypes):
```

Comment:
how is this error different from the previous one? and from the following one? could we not just have the one on line 29 and remove the following 2 on lines 36 and 43?
In my head they are the same, I don't understand the difference. If they are different, could you please add more info to help the reviewer understand why this tests are important? what function of the code you wrote in feature_selection.py are you testing?